README.md

```diff
@@ -30,4 +30,9 @@ yarn
 ./regen_openapi.sh https://api.svix.com/api/v1/openapi.json
 ```
 
+```sh
+# remove generated files
+git clean -xdf
```

Comment:
```suggestion
git clean -Xdf
```
You probably want this (`-X` rather than `-x`), in order to avoid accidentally deleting files that were manually created. Though tbh this is just a generic git tutorial and probably doesn't belong here.
Especially worrisome is that it's such a foot-gun.